src/pages/LoginPage.js

```diff
@@ -1,9 +1,44 @@
 import React from 'react'
+import {ThemeButton} from "../components/ThemeButton";
+import { useForm } from "react-hook-form";
+import {TextField, Container, Grid, Box, Link} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import { Link as RouterLink } from 'react-router-dom';
+import {login} from "../mock/auth";
 
 export const LoginPage = () => {
+    const { register, handleSubmit, errors } = useForm();
+    const onSubmit = data => {
+        console.log(data);
+        login();
+    }
+    const classes = useStyles();
+
   return (
-    <div>
-      LoginPage
-    </div>
+      <Container className={classes.container} maxWidth="xs">
+          <form onSubmit={handleSubmit(onSubmit)}>
+              <Grid container
+                    direction="column"
+                    justify="space-between"
+                    alignItems="center"
+              >
+                  <Box><TextField label="Username" name="usernameLoginField" inputRef={register( {required: true})} /></Box>
+                  <Box marginTop={2}><TextField label="Password" name="passwordLoginField" inputRef={register({ required: true })} type="password" /></Box>
+                  <Box marginTop={5}><ThemeButton text={"log in"}/></Box>
```

Comment:
Kanskje det går an å legge inn en `<CircularProgress />` på knappen når man prøver å logge inn og venter på responsen. Går an å legge til `formstate: { isSubmitting }` i `useForm()` for å sjekke om submit funksjonen er ferdig.

Reply:
Åh, så kult! Må innrømme at jeg ikke skjønte helt hva CircularProgress var før jeg googlet det nå. Så for meg at det var noe veldig komplisert noe når jeg så det i koden din haha. Men ja, skal prøve å fikse det! 😎